src/searchForm/autoComplete/section/index.tsx

```diff
@@ -6,37 +6,37 @@ import autoComplete from './AutoCompleteSection'
 const StyledAutoComplete = styled(autoComplete)`
   & {
     box-sizing: border-box;
-  }
-
-  & .kirk-textField-wrapper {
-    background: transparent;
-    border-radius: 0;
-    border: 0;
-  }
-
-  & .kirk-textField-wrapper input {
-    font-size: ${font.m.size}
-    line-height: ${font.m.lineHeight}
-    background: transparent;
-    border-radius: 0;
-  }
-
-  & .kirk-textField-wrapper .kirk-loader {
-    position: absolute;
-    top: 12px;
-    right: 12px;
-  }
-
-  & .kirk-items-list {
-    padding: 0;
-  }
-
-  & .kirk-items-list.kirk-autoComplete-list .kirk-items-list-item::before {
-    content: none;
-  }
 
-  & .kirk-items-list .kirk-items-list-item .kirk-item-choice {
-    padding: ${space.m} ${space.l};
+    .kirk-textField-wrapper {
+      background: transparent;
+      border-radius: 0;
+      border: 0;
+
+      input {
+        font-size: ${font.m.size}
+        line-height: ${font.m.lineHeight}
+        background: transparent;
+        border-radius: 0;
+      }
+
+      .kirk-loader {
+        position: absolute;
+        top: 12px;
```

Comment:
please, use **spaces** defined on utils/branding

Reply:
12 px does not exist in the branding, you will end up adding constants to create a new const that has no semantic value.
If the PR is about adding nesting, I would keep it focused on just that.

Reply:
I mean.. if its a boyscout we should care about the overal style. 
Indeed `12px` doesn't exists, and that is the point: remove **magical numbers**.

Reply:
I agree with @grosbouddha, since the PR is about CSS nesting we should only focus on the PR topic and not change anything else but we can plan another PR to fix this issue

Reply:
Even if it's a boyscout PR, it's good to have only one goal per PR, here the goal was to add nesting, not fixing magic numbers.